exercises/sum-of-multiples/description.md

```diff
@@ -5,3 +5,5 @@ If we list all the natural numbers up to but not including 20 that are
 multiples of either 3 or 5, we get 3, 5, 6 and 9, 10, 12, 15, and 18.
 
 The sum of these multiples is 78.
+
+PS: Notice that multiples of both 3 and 5 like for example 15 will counted only once. 
```

Comment:
This post scriptum feels out of place (well, thats what post scriptums always do).

I'd prefer to add the word "unique" as a qualifier to the first sentence:

> Given a number, find the sum of all the **unique** multiples of particular numbers up to
but not including that number.

(emphasis to make the difference clear, not to be added into the description)